src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -226,10 +226,13 @@ export const MousePopulationsModel = types
 
           buttons.push({
             title: "Scale",
+            type: "float-button",
             value: self.showMaxPoints,
             action: (val: boolean) => {
               self.toggleShowMaxPoints();
-            }
+            },
+            floatCorner: "upper-right",
+            section: "data"
           });
```

Comment:
Use `floatCorner` to specify where we will position the button (for now I'm thinking of the buttons as floating in one of the 4 corners, but this could be made more generic for things like the top-center, bottom-center, etc.)  `section` specifies which of the 3 right panel sections shows the buttons.  I'm assuming each section might have it's own set of toolbar buttons